test cases/common/14 configure file/check_inputs.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+import sys
+from pathlib import Path
+
+files = [Path(f) for f in sys.argv[1:]]
+names = [f.name for f in files]
+
+assert(names == ['check_inputs.txt', 'prog.c', 'prog.c', 'prog2.c', 'prog4.c', 'prog5.c'])
```

Comment:
tiny nit, `assert` is a keyword, don't use parens with it. It is really actively harmful to use them because:
```python
assert a == 0, 'something should have happened that didnt'
```
is *not* the same as:
```python
assert(a == 0, 'something should have happened that didnt')
```

in the latter case what you're actually asserting is `tuple(T, U)`, which will *always* be true.

Reply:
ok, didn't know that. Thanks.